qtconsole/completion_widget.py

```diff
@@ -20,9 +21,28 @@ def __init__(self, console_widget):
         super(CompletionWidget, self).__init__()
 
         self._text_edit = text_edit
+        self.setEditTriggers(QtGui.QAbstractItemView.NoEditTriggers)
+        self.setHorizontalScrollBarPolicy(QtCore.Qt.ScrollBarAlwaysOff)
+        self.setSelectionBehavior(QtGui.QAbstractItemView.SelectRows)
+        self.setSelectionMode(QtGui.QAbstractItemView.SingleSelection)
+
+        # Use platform specific window flags to ensure correct behavior
+        # See github.com/jupyter/qtconsole/pull/93 for detailed discussion
+        if os.name == 'nt':
+            # On windows, we need Popup style to ensure correct mouse
+            # interaction (dialog would dissappear on mouse click with
+            # ToolTip style)
+            flags = QtCore.Qt.Popup
+        else:
+            # On other OS, we need ToolTip style to ensure that the widget
+            # shows correctly (dialog is not drawn with Popup style)
+            flags = QtCore.Qt.ToolTip | QtCore.Qt.WindowStaysOnTopHint
 
         self.setAttribute(QtCore.Qt.WA_StaticContents)
-        self.setWindowFlags(QtCore.Qt.ToolTip | QtCore.Qt.WindowStaysOnTopHint)
+        origPolicy = text_edit.focusPolicy()
```

Comment:
Please change this to `original_policy` to follow the Python custom of naming variables with underscores instead of camelcase.